target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -525,6 +525,7 @@ ramips_setup_macs()
 	arcwireless,freestation5|\
 	lenovo,newifi-y1|\
 	lenovo,newifi-y1s|\
+	netgear,ex3700|\
```

Comment:
This is set up as one-port, too. So the same question concerning WAN MAC address arises...

Reply:
Commit removed